server/db/index.js

```diff
@@ -0,0 +1,6 @@
+var mongoose = require('mongoose');
+
+const db = mongoose.connect('mongodb://localhost/listings', { useNewUrlParser: true, useUnifiedTopology: true });
+
+
+module.exports = db;
```

Comment:
per style guidelines, make sure to remember a new line at the end of the document, this one has gotten me a few times already

Reply:
yeah, my eslint isn't working.  I will fix tomorrow to catch issues like this.

Its a real weird bug too, it has a problem with azure services for some reason.  I might just remove it all and start fresh.